pkgs/applications/version-management/gogs/default.nix.patch

```diff
@@ -0,0 +1,22 @@
+2,3c2
```

Comment:
I tried to separate ``default.nix`` from generated one in https://github.com/kamilchm/go2nix/pull/10 but it looks like the ``allowGoReference`` flag doesn't work :/
Is there a better way to override generated file than ``import``?